dashboards/resource_lakeview_dashboard.go

```diff
@@ -0,0 +1,179 @@
+package dashboards
+
+import (
+	"context"
+	"crypto/md5"
+	"fmt"
+	"log"
+
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Calculates MD5 hash of the given content
+func calculateMd5Hash(content []byte) string {
+	return fmt.Sprintf("%x", md5.Sum(content))
+}
```

Comment:
We could move it to common/util.go similar to what we did for ReadFileContents since it's more general in nature.